miscellaneous/metaDataRemover/config.toml

```diff
@@ -0,0 +1,5 @@
+ProxyUrl = ""
+
+TokensToDeletePerTransaction = 0
```

Comment:
why 0?

Reply:
Wanted to leave all values to some default values.
However, changed `TokensToDeletePerTransaction` && `GasLimit`